bodhi/server/models.py

```diff
@@ -380,6 +380,16 @@ class UpdateStatus(DeclEnum):
     processing = 'processing', 'processing'
 
 
+class CiStatus(DeclEnum):
+    """ This class lists the different status the ci_status flag can have. """
+    ignored = 'ignored', 'Ignored'
+    queued = 'queued', 'Queued'
+    running = 'running', 'Running'
+    passed = 'passed', 'Passed'
+    failed = 'failed', 'Failed'
+    waiting = 'waiting', 'Waiting'
```

Comment:
Could we put some comments over these to explain what each one means? I think the comments you had in that fpaste recently would be perfect here.